pages/index.tsx

```diff
@@ -144,77 +274,258 @@ export default function Home() {
+              </div>
+            </div>
+          ))}
+        </div>
+        <div className={styles.testArea} id="testArea">
+          <div className={styles.withAccelerate}>
+            <h3>
+              <img src="/bolt.svg" /> With Accelerate Cache
+            </h3>
+            <div className={styles.illustrationSection}>
+              <Animation
+                autoplay={false}
+                state={state}
+                name="with-accelerate"
+                className={styles.animation}
+                fit={reactCanvas.Fit.FitWidth}
+                badge={
+                  <span
+                    className={styles.badgeYellow}
+                    style={{ top: "-30px", left: "6%" }}
+                  >
+                    <img
+                      src="/locationDot.svg"
+                      color={defaultTheme.colors.orange[400]}
+                      width="8px"
+                      height="15px"
+                    />
+                    {history?.[0]?.location ?? ""}
+                  </span>
+                }
+                badge2={
+                  <span
+                    className={styles.badgeYellow}
+                    style={{ top: "-30px", left: "35%" }}
+                  >
+                    <img
+                      src="/locationDot.svg"
+                      color={defaultTheme.colors.orange[400]}
+                      width="8px"
+                      height="15px"
+                    />
+                    {history?.[0]?.location ?? ""}
+                  </span>
+                }
+                badgeVirginia={
+                  <span className={`${styles.badgeYellow} ${styles.virginia}`}>
+                    <img
+                      src="/locationDot.svg"
+                      color={defaultTheme.colors.orange[400]}
+                      width="8px"
+                      height="15px"
+                    />
+                    n.virginia
+                  </span>
+                }
+              />
+            </div>
+            <div className={styles.cardInfo}>
+              <div className={styles.numbers}>
+                <h4>{num.format((1_000 / cacheLatency) * 60)}</h4>
+                <span>queries per minute</span>
+              </div>
+              <div className={styles.numbers}>
+                <h4>{ms.format(cacheLatency)}</h4>
+                <span>latency</span>
+              </div>
+              <p>
+                The result of the database query is cached at the Accelerate
+                caching node in&nbsp;
+                <span className={styles.badgeYellow}>
+                  <img
+                    src="/locationDot.svg"
+                    color={defaultTheme.colors.orange[400]}
+                    width="8px"
+                    height="15px"
+                  />
+                  {history?.[0]?.location ?? ""}
+                </span>
+                &nbsp; and retrieved from there:
```

Comment:
```suggestion
                &nbsp;and retrieved from there:
```

otherwise a double space is rendered